site/docs/components/overview.mdx

```diff
@@ -27,6 +27,23 @@ We strive to achieve the following quality attributes on all of our components:
 
 The status of these can be found on the documentation page for each component, or on the [Component health](/component-health) page, which shows the status of all components.
 
+### API
+- **Avoid language overloading**: Avoid prop names that are used in HTML, such as `type`.
```

Comment:
```suggestion
- **Avoid language overloading**: Choose names that are self-describing that continue to be clear in slightly different contexts. For example, `type`, has a specific meaning within the context of a form, but it could also be used to describe a different appearance of a component. A good rule of thumb is to avoid names that are used in HTML.
```